src/main/java/alexiil/mc/lib/attributes/fluid/compat/mod/LbaFluidModCompatLoader.java

```diff
@@ -26,6 +26,7 @@ public static void load() {
             LibBlockAttributes.LOGGER.info("Silk not found, not loading compatibility for fluids.");
         }
 
-        RebornCompatLoader.load();
+        //FIXME: Fix reborn core compat once it updates to 1.17
+        //RebornCompatLoader.load();
```

Comment:
Reborn core compat might be better off excluded from the build completely - it might be better to add `exclude "alexiil/mc/lib/attributes/fluid/compat/mod/reborncore/**"` to the source sets block (like this) in build.gradle:
```gradle
sourceSets {
    main {
        java {
            srcDir "src/main/java"
            srcDir "src/fatjar/java"
            exclude "alexiil/mc/lib/attributes/fluid/compat/mod/reborncore/**"
        }
    }
}
```